packages/framework/ai-collab/src/aiCollab.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ImplicitFieldSchema } from "@fluidframework/tree";
+
+import type {
+	AiCollabErrorResponse,
+	AiCollabOptions,
+	AiCollabSuccessResponse,
+} from "./aiCollabApi.js";
+import { generateTreeEdits } from "./explicit-strategy/index.js";
+
+/**
+ * Calls an LLM to modify the provided SharedTree based on the provided users input.
```

Comment:
Since this is the main entrypoint to the library, it would probably be good to expand the docs here. In particular, I would recommend:
- An `@example` comment block or two, highlighting how to use this function. We probably don't need quite as much detail as the README, but some very basic examples showing how the API could be called to achieve different outcomes would be useful.
- This seems like an appropriate place for us to document limitations of our system. Based on our last high-level discussion on the topic, I believe the intention is for this to be a starting place, and that more complex usages will likely need to implement their own strategy/strategies. I would be interested in thoughts from @kashms and @rohandubal on how we may want to document this, but I do think we need to cover ourselves from getting bug reports for LLM behaviors and such where we don't intend to make guarantees.

Reply:
The documentation here has been updated, including an example block as of commit e835b1c037abfbab72df0c82e66781f54a4bdfe9